content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,22 @@
+---
+title: IMDSv2 Enablement by Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+With the release of Agent v7.64.0, Datadog defaults to using IMDSv2 for enhanced security and consistency in retrieving AWS metadata.
+If your host has already configured the `ec2_prefer_imdsv2` flag to `true`, you won't notice any changes as IMDSv2 is already enabled.
+
+If you upgrade to Agent v7.64.0 without any custom configuration changes, you may see a change to how hostnames appear in Datadog. After this update, Datadog displays hostnames as instance IDs in Fleet Automation and on the Infrastructure list.
+This change only impacts how hostnames are displayed. Metric tagging continues to use your original hostnames. Monitoring, alerting, and dashboards are not affected.
+
+### Canonical Hostname
+
+The hostname picked to tag metrics is named the `canonical hostname`, this hostname is the unique ID used to identify your host. Even if your hostname change after upgrading to v7.64.0+, the `canonical hostname` of your host will not change, you can continue to refer to the previous one as before.
```

Comment:
```suggestion
The hostname picked to tag metrics is named the _canonical hostname_. This hostname is the unique ID used to identify your host. Even if your hostname changes after upgrading to v7.64.0+, the `canonical hostname` of your host does not change, and you can continue to refer to the previous hostname.
```

Reply:
I'd also add `picked to tag metrics` -> `chosen to associate tags with metrics`